openapi/schemas/processes-core/statusInfo.yaml

```diff
@@ -11,12 +11,24 @@ allOf:
       processID:
         type: string
         format: uri
-      type:
-        type: string
-        example:
-          - process
-          - wps
-          - openeo
+      processingEntityType:
+        description: |-
+          The type of entity that created the job and is doing the processing.
+          This includes all the data access apis listed in "apis.yaml" plus
+          the processing APIs of OGC API Processes and OpenEO.
+        allOf:
+          - $ref: "apis.yaml"
+          - type: string
+            enum:
+              - ogc-api-processes
+              - open-eo
```

Comment:
I think this will fail all the time. An `processingEntityType` value can never succeed both `enum` clauses simultaneously. Should be `oneOf` or `anyOf`. 

Small nitpick. I think `openeo` is better since that is how it is written everywhere. 
- OGC API Processes => `ogc-api-processes`
- openEO => `openeo`

Reply:
Fixed.